.github/workflows/docker.yml

```diff
@@ -0,0 +1,84 @@
+on:
+  schedule:
+    - cron: '0 7 */7 * *' # This should run once a week
+
+name: Docker images
+
+jobs:
+  build-images:
+    name: Build & push Docker images
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        runtime:
+          - name: fdo-20-08
+            platform: org.freedesktop.Platform//20.08
```

Comment:
Installing the sdk installs the platform, no?

Reply:
No (which I think is weird). You need to manually install both